components/chainhook-sdk/src/chainhooks/stacks/mod.rs

```diff
@@ -544,6 +544,7 @@ fn serialize_stacks_transaction(
                 json!(transaction.metadata.result)
             },
             "sender": transaction.metadata.sender,
+            "nonce": transaction.metadata.nonce,
```

Comment:
nice; glad it was as straightforward as this!

I see references to ["hex string"](https://github.com/hirosystems/chainhook/blob/ec76f29dd95818f2645f04d61ebaf3a8af1bad87/components/chainhook-types-rs/src/rosetta.rs#L19), so how do we feel about this feedback/ask: https://github.com/hirosystems/chainhook/issues/422#issuecomment-1742285097?

Reply:
We need further clarification on that. I can work it in a separate PR once we have the info we need. Also, I've created a separate issue to track it: #431